### DISCUSSION TITLE
Redis default password?

### DISCUSSION THREAD

**Discussion by simplenotezy** [coollabsio/coolify#3983] [Q&A] [answered]
I've tested out creating a Redis instance, but I don't know what the password is. I ssh'ed into the machine ran redis-cli, but when issuing `PING` I get `NOAUTH Authentication required.`

I did not configure a redis password; what's the default one?

**Comment by peaklabs-dev** [comment, marked as answer]
You can see the default password in the input field and the `REDIS_PASSWORD` environment variable. As soon as the next version is available (I fixed it). For now you can have a look at the Redis URL which includes the password.
